api/src/test/java/com/epam/pipeline/manager/pipeline/FolderTemplateManagerTest.java

```diff
@@ -75,70 +62,27 @@
     private static final String CHILD_TEMPLATE_FOLDER_NAME_1 = "test-1";
     private static final String DATASTORAGE_NAME_1 = "ds-1";
 
-    @Autowired
+    @SpyBean
```

Comment:
Probably there is no need for `SpyBean` here. `Autowired` would be enough.

Reply:
Fixed